prow/gen-config.sh

```diff
@@ -10,6 +15,7 @@ echo "#======================================
 #    Run gen-config.sh to regenerate.
 #======================================" > config.gen.yaml
 
-for file in config/*; do
-  cat "${file}" >> config.gen.yaml
+for file in "${DIR}"/config/*; do
+  # shellcheck disable=SC2016 ## in case of sed expression first '' is not an actual variable to be expanded
+  sed -e 's@${NAMESPACE}@'"${NAMESPACE}"'@' -e 's@${WORKER_NS}@'"${WORKER_NS}"'@' "${file}" >> "${DIR}"/config.gen.yaml
```

Comment:
I'd put a `g` at the end of sed `s` command

Reply:
Good idea! Thank you. Added in 75ea8f3